extensions/cornerstone/src/panels/PanelMeasurement.tsx

```diff
@@ -48,12 +51,25 @@ export default function PanelMeasurementTable({
 
   const additionalFilter = filterAdditionalFinding(measurementService);
 
-  const { measurementFilter: trackedFilter } = measurementFilters;
-  const measurements = displayMeasurements.filter(
-    item => !additionalFilter(item) && trackedFilter(item)
-  );
+  const { measurementFilter } = measurementFilters;
+  const measurements = displayMeasurements
```

Comment:
This grouping function is basically what should be done, but the point is to do it generically with a passed in grouping method so that one can group by study, but also in the future use other grouping methods.   Also, it should be recursive, so that the child grouping items are also using the same group.